stores/metadata.go

```diff
@@ -1364,6 +1412,11 @@ func (ss *SQLStore) UpdateSlab(ctx context.Context, s object.Slab, contractSet s
 			return err
 		}
 
+		// invalidate health
+		if err := tx.Model(&slab).Where(&slab).Update("health_valid", false).Error; err != nil {
```

Comment:
Same here, would be neat to try this with a `gorm` hook. Not because it's cool but also because it's easy to forget to invalidate health. Cache invalidations through hooks are always idea imo